post_office/mail.py

```diff
@@ -270,7 +272,26 @@ def send(email):
     Email.objects.filter(id__in=email_ids).update(status=STATUS.sent)
 
     email_ids = [email.id for (email, e) in failed_emails]
-    Email.objects.filter(id__in=email_ids).update(status=STATUS.failed)
+
+    numbers_mails_get_back_in_queue = 0
```

Comment:
Can we change this to `requeued_emails`?

Reply:
ok